install.sh

```diff
@@ -180,10 +178,6 @@ while (($# > 0)); do
       INSTALL_PREFIX="$2"
       shift
       ;;
-    --base-name | -b)
```

Comment:
@kwlzn I got rid of this because I assume you just copy-pastaed from scie-pants. In that case there was a period of trial where the pants bash script was still in use where having a non-standard name was useful. We don't have that transition problem; so this knob seems gratuitous to me.

Reply:
yep - sgtm.